### PR TITLE
Fix config import.

### PR DESCRIPTION
**Pull request by xals**

### CODE REVIEW

**Comment by xals** [comment]
I included a fix on "connect".

File: addok_psql/processors.py [hunk 1]
```diff
@@ -6,11 +6,17 @@
 import psycopg2
 import psycopg2.extras
 
-from addok import config
+from addok.config import config
 from addok.helpers import yielder
 
 
-connection = psycopg2.connect(**config.PSQL)
+connection = psycopg2.connect(
```
Comment (yohanboniface) [inline addok_psql/processors.py]: What's the rationale behind this change? Doing so will prevent using any specific [connect](http://initd.org/psycopg/docs/module.html?highlight=connect#psycopg2.connect) argument afaik

Comment (xals) [reply addok_psql/processors.py]: I wanted to keep parameter names from config unchanged, and the kwargs in connect have not the same name as described there: http://initd.org/psycopg/docs/module.html

Comment (xals) [reply addok_psql/processors.py]: (actually before this change I had an error on 'dbuser' kwarg which does not exist in connect)

Comment (xals) [reply addok_psql/processors.py]: Anyway, we can just keep **config.PSQL and change config parameter names if you prefer.

Comment (xals) [reply addok_psql/processors.py]: Hm. I may be wrong on this one, it seems my parameters are not correctly handled.

Comment (xals) [reply addok_psql/processors.py]: Ok, will delay it for another PR, after better testing.

**Comment by xals** [comment]
Sorry for the noise, I was not on the right branch.
